### DISCUSSION TITLE
Color tokens - Unused and inconsistencies

### DISCUSSION THREAD

**Discussion by aweell** [Telefonica/mistica-design#596] [Proposals]
# Tokens name

* [movistarBlue20](https://github.com/Telefonica/mistica-design/discussions/596#discussioncomment-3107815)
* [movistarBlue30](https://github.com/Telefonica/mistica-design/discussions/596#discussioncomment-3120273)

**Comment by aweell** [comment]
**Name: `movistarBlue20`**
**Brand: Movistar**

### Constant used in:
* `toggleAndroidBackgroundActive` 
* `textNavigationBarSecondary` 
* `textNavigationSearchBarHint`

### Token usage

**Figma**

| token name | usage | location |
| :----- | :------ | :----- |
| `toggleAndroidBackgroundActive` |✅ | `switch[D]`, `Android/Switch`, `Web/Switch` |
| `textNavigationBarSecondary` | ⁉️ | none |
| `textNavigationSearchBarHint` |⁉️ | none |

**Web**

| token name | usage | location |
| :----- | :------ | :----- |
| `toggleAndroidBackgroundActive` |✅ | `switch-component.tsx` |
| `textNavigationBarSecondary` | ⁉️ | none |
| `textNavigationSearchBarHint` |⁉️ | none |

**iOS**

| token name | usage | location |
| :----- | :------ | :----- |
| `toggleAndroidBackgroundActive` | N/A | - |
| `textNavigationBarSecondary` | ✅ | `UIBarButtonItem+Utils.swift` |
| `textNavigationSearchBarHint` |⁉️ | NONE |

**Android**

| token name | usage | location |
| :----- | :------ | :----- |
| `toggleAndroidBackgroundActive` | ⁉️ | none |
| `textNavigationBarSecondary` | ⁉️ | Doesn't seems to be used |
| `textNavigationSearchBarHint` |⁉️ | none |


## Updates needed

- [ ] Remove `textNavigationSearchBarHint`

**Comment by aweell** [comment]
Name: **`movistarBlue30`**
Brand: Movistar

### Constant used in:

* `buttonPrimaryBackgroundSelectedInverse`
* `buttonSecondaryBorderSelectedInverse`
* `textLinkSnackbar`
* `loadingBar`


### Token usage

**Figma**

| token name | usage (D / m) | location |
| :----- | :------ | :----- |
| `buttonPrimaryBackgroundSelectedInverse` | ⁉️ / ✅ | `Buttons [D]`, `Buttons` |
| `buttonSecondaryBorderSelectedInverse` | ✅ / ✅ | `Buttons [D]`, `Buttons` |
| `textLinkSnackbar` |✅ / ✅ | `iOS/Crouton`, `Android/Snackbar`, Web/Snackbar |
| `loadingBar` | ✅ / ✅ | `loadingBar` |

#### Notes

* For the `inverse`, `selected`, `small` variant of the button primary in `desktop`, `buttonLinkBackgroundSelected` (`movistarBlue10`) it's been used instead as `background-color`.

**Reply by yceballost** [reply]
that's a bug

fixed: https://github.com/Telefonica/mistica-design/issues/608
